app/controllers/api/v1/users_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::V1::UsersController < ApplicationController
   def create
-    user = User.create(user_params)
+    user = User.new(user_params)
```

Comment:
Not sure if this makes a difference, but a colleague suggested this over create to help troubleshoot an issue with Postman being unable to hit this end point. I've got it working now, but I'm fairly sure it was due to additional headers that I wasn't aware of that needed to be included with the request.